course/src/main/java/org/openedx/course/presentation/ui/CourseUI.kt

```diff
@@ -292,6 +295,77 @@ fun CourseSectionCard(
     }
 }
 
+@Composable
+fun OfflineQueueCard(
+    downloadModel: DownloadModel,
+    progressValue: Long,
+    progressSize: Long,
+    onDownloadClick: (DownloadModel) -> Unit
+) {
```

Comment:
Could you please add a preview for this Card?

Reply:
done